README.md

```diff
@@ -135,6 +135,24 @@ See [routesapi module](http://github.com/gliderlabs/logspout/blob/master/routesa
 
 Logspout relies on the Docker API to retrieve container logs. A failure in the API may cause a log stream to hang. Logspout can detect and restart inactive Docker log streams. Use the environment variable `INACTIVITY_TIMEOUT` to enable this feature. E.g.: `INACTIVITY_TIMEOUT=1m` for a 1-minute threshold.
 
+#### Multiline logging
+
+In order to enable multiline logging, you must first prefix your adapter with the multiline adapter:
+
+	$ docker run \
+		--volume=/var/run/docker.sock:/var/run/docker.sock \
+		gliderlabs/logspout \
+		multiline+raw://192.168.10.10:5000?filter.name=*_db
+
+Multiline logging is enabled by default on all containers. To enable it only to specific containers set MULTILINE_ENABLE_DEFAULT=false for logspout, and use the LOGSPOUT_MULTILINE environment variable on the monitored container:
```

Comment:
Can you clarify this to say that its only enabled by default if you specify the `multiline+` prefix?